### ISSUE TITLE
Tool Profiles don't "Take" when assigned from the menus.

### ISSUE THREAD

**Issue by MystLeissa** [LecrisUT/Rimworld-SurvivalTools-old#53]
I've run into a issue with Tools not being "picked up" automatically by colonists. This also results in them not wanting to execute actions such as "Cut Trees" when I order them to. I think it's possibly a mod incompatibility but I have no Idea which one.

The Mods I use:

*Libs/Frameworks*

Harmony
https://steamcommunity.com/sharedfiles/filedetails/?id=2009463077

Toolbox
https://steamcommunity.com/sharedfiles/filedetails/?id=2016186459

HugsLib
https://steamcommunity.com/sharedfiles/filedetails/?id=818773962

*Modifiers*

[LC] Survival Tools (this mod)
https://steamcommunity.com/sharedfiles/filedetails/?id=2057841936

Trade Helper
https://steamcommunity.com/sharedfiles/filedetails/?id=2113372560

Restricted Storage:
https://steamcommunity.com/sharedfiles/filedetails/?id=2015159617

Surgical Body Shaping (SBS)
https://steamcommunity.com/sharedfiles/filedetails/?id=1217321550

*Misc Items/Constructions*

Fishing Nets
https://steamcommunity.com/sharedfiles/filedetails/?id=1529981982

Dye Vats:
https://steamcommunity.com/sharedfiles/filedetails/?id=740002420

BondageBed Torture
https://steamcommunity.com/sharedfiles/filedetails/?id=2094933315

Architect Expanded - Fences
https://steamcommunity.com/sharedfiles/filedetails/?id=2050680665

[SS]Maid Project
https://steamcommunity.com/sharedfiles/filedetails/?id=1498756997

*Disabled but Subbed*

Empire
https://steamcommunity.com/sharedfiles/filedetails/?id=2006535182

**Comment by LecrisUT** [comment]
I'll be reworking the tool optimizing method for the next update, and also include an opportunistic picking of tools. In the meantime do you get any errors in the log?

**Comment by MystLeissa** [comment]
Where do I find the logs? I'm kinda new to RimWorld...

**Comment by MystLeissa** [comment]
if you mean the debug.log it says this:
0630/155702.839:ERROR:settings.cc(325)] Settings version is not 1
[0630/192901.292:ERROR:settings.cc(325)] Settings version is not 1
[0630/214152.011:ERROR:settings.cc(325)] Settings version is not 1
and that's it - not much to go on....

**Comment by LecrisUT** [comment]
Ctrl+F12 and follow the instructions there to upload the log, this is commonly referred to as the HugsLog or similar names. The game's log can be viewed by enabling developer mode and it's one of the buttons on the top.

**Comment by MystLeissa** [comment]
https://gist.github.com/HugsLibRecordKeeper/cbe7a9de42e8d7a9505bce965a6cc639

I hope i did that right...

**Comment by LecrisUT** [comment]
Yep that's good. Now just do that when the tool should be picked up. You can follow the dev log and see if any red errors appear and upload once it does

**Comment by MystLeissa** [comment]
I /think/ I managed to identify one possible incompatibilty, I read on Toolbox's mod incompatibilities that things that alter ThingDefs are somewhat incompatible...I may have to lose my fences as a result (since Toolbox is only loaded for fences)

**Comment by MystLeissa** [comment]
Only thing I see in the logs is a yellow error about missing sound files: https://gist.github.com/HugsLibRecordKeeper/33e6bed62ab03760b9dd82382ce3b688

**Comment by LecrisUT** [comment]
It's not a incompatibility with toolbox, but I'll double check when I've got time.

**Comment by MystLeissa** [comment]
Thanks your help is apprieciated. I'll just deal with it the old fashioned way for now - they /will/ pick them up if i force them to equip them - it's just annoying because the tool profiles don't automatically select what I want...
